modules/ROOT/pages/create-profiles.adoc

```diff
@@ -72,61 +72,90 @@ If you filter by categories, you can see the number of categories for each API,
 |
 Set Notifications
-|Enable or disable automatic notifications for this profile. The default is enabled. See xref:configure-notifications.adoc#send-auto-notifs[Sending Conformance Notifications].
+|Enable or disable automatic notifications for this profile. The default setting is enabled for active profiles. Notifications are automatically disabled for draft profiles. See xref:configure-notifications.adoc#send-auto-notifs[Sending Conformance Notifications].
 |Specify General Information 
 |Enter a profile name and description that helps others understand the kinds of governance rulesets included and the kinds of APIs validated in this profile.
 |Review
-|Review your profile and click *Create*.
+|Review your profile and click *Save as a draft* or *Save as active*:
+
+* Save a profile as draft to test profile settings and review conformance information before revealing the conformance information for its targeted APIs in integrating Anypoint Platform products.
+* Save a profile as active to reveal the conformance information for its targeted APIs in integrating Anypoint Platform products.
+
+When you save a profile as either draft or active, its targeted APIs become governed.
 |
 |===
 
 == Update a Governance Profile Using the API Governance Console
 
-The API Governance console guides you through updating profiles. 
+The API Governance console UI guides you through updating profiles. 
 
-To update governance profile information using the API Governance console: 
+To update governance profile information using the console: 
 
 . In *API Governance*, select the console *Profiles* tab.
+. Click *Draft* or *Active*.
 . For the profile you want to update, click the more options menu icon (image:overflow-menu-icon.png[width=9,height=15,fit=line]) and then select *Edit*.
 . Navigate through the UI using the *Next* and *Previous* buttons and update the profile information as needed. For details on the options, see <<create-profile-in-console>>.
 . Review your changes and click *Update Profile*.
-
-NOTE: After you apply your edits, the total APIs are updated to indicate how many APIs
++
+After you apply your edits, the total APIs value refreshes to indicate how many APIs
 your governance profile includes based on your updated filter criteria.
 
-== Delete a Governance Profile Using the API Governance Console
+[[activate-draft-profile]]
+== Activate a Draft Governance Profile
+
+To activate a draft profile: 
 
-The API Governance console guides you through deleting profiles.
+. In *API Governance*, select the console *Profiles* tab.
+. Click *Draft* to show draft profiles.
+. Click  *Activate* for the profile you want to activate.
++
+After you activate a profile, the APIs validated against the rulesets in that profile become governed APIs and their conformance information appears in integrating products in Anypoint Platform.
```

Comment:
I guess "integrating products" is an area in the platform.

Reply:
What do we call this as a standard? In some places I say "across Anypoint Platform", and don't want to list the integrating products each time, like Exchange, Design Center, and API Manager now and soon to be ACB. Do you like that better?

Reply:
As in the other comment, I've changed it to "outside of the draft profile view."

Reply:
I wonder if "integration products" is more clear? In your first mention, you could say "...products, such as Exchange, yadda, yadda" and subsequently use the shorter name.

Reply:
The doc already has information about Exchange, etc. in the Overview and lots of other places. It's not "integration products", so that wouldn't work. I prefer to keep it like this for now, as I think it is understandable. The draft profile feature is just very confusing and they changed the behavior late in the release after we'd chosen all the terminology in many meetings with many people involved. Long story short, I'm having to compensate in the doc for a confusing feature. Thanks for being so diligent in trying to help clarify. I think this helps, and I'll see if Eng and the PM have feedback as well.